### DISCUSSION TITLE
Ubuntu server, no Docker, auto update possible?

### DISCUSSION THREAD

**Discussion by HubKing** [TriliumNext/Trilium#2286] [Q&A] [answered]
Almost all other applications are installed using the standard `apt install` method, and they can be automatically installed using Ubuntu's software update. Now, if I install Trilium with the Wiki section's "Packaged server/manual installation" method, is Trilium automatically updated? I kind of think it would not be. 

What is the installation method for allowing automatic update? Do I have to use the Docker method for this?

**Comment by zadam** [comment, marked as answer]
Hi, there's no auto update. Docker is not auto updating either.

**Reply by HubKing** [reply]
Any plan for supporting automatic updates in the foreseeable future?

**Comment by sigaloid** [comment]
You could possibly set up a bash script to download it and overwrite the old version, then start the new one. Then have systemd run the script every x hours/days.

```bash
curl -L $(curl --silent https://api.github.com/repos/zadam/trilium/releases/latest | jq .assets[1].browser_download_url | sed -r 's/"//g') --output "trilium.tar.gz"
```
This oneliner will download the latest `trilium-linux-x64-server-$(V).tar.xz` from the Releases tab. Finish the script to download it and extract (and overwrite) existing folder, then restart the server. 

(You need to run `sudo apt install jq curl sed` before this script will work)
